docs/data-integrations/firebird.mdx

```diff
@@ -13,10 +13,10 @@ This handler is implemented using the `fdb` library, the Python driver for Fireb
 
 The required arguments to establish a connection are as follows:
 
-* `host` is the host name or IP address of the Firebird server.
-* `database` is the port to use when connecting with the Firebird server.
-* `user` is the username to authenticate the user with the Firebird server.
-* `password` is the password to authenticate the user with the Firebird server.
+- `host` is the host name or IP address of the Firebird server.
```

Comment:
You changed `*` into `-`.

Please change it back to `*` for all four points.

Reply:
For more clarification do I have to change the 16th point + - to  + *